feedback.md

```diff
@@ -0,0 +1,25 @@
+# info343-final-project
+
+Our team is proposing the creation of a React-based trivia game web app. We will use one of the available trivia question APIs to create a web-based game with the following functionality:
+
+- User accounts, managed by Google Firebase
+- Sign-in/Sign-out
+- API manipulation, to pull down questions, using the Open Trivia DB (opentdb.com)
+- A timer (10 seconds to answer each question, for example)
+- Leaderboard rankings
```

Comment:
graphs/charts?

Reply:
We could explore some sort of graphical representation of the results, like a bar whose length represents the number of correct questions.

Reply:
how will you make these charts?
There are great packages/libraries that help you with this such as: d3.js, plotly.js, chart.js

Reply:
I think we will going to use chart.js